src/search.c

```diff
@@ -4358,6 +4364,10 @@ fuzzy_match_compute_score(
 	    score += FIRST_LETTER_BONUS;
 	}
     }
+ #if 0
```

Comment:
Hi Christian,

Can you add a comment here about why this code is commented out?

Thanks,
Yegappan